### PR TITLE
Add dapr docs

### PR DESCRIPTION
**Pull request by greenie-msft**
# Description

Initial PR for adding Dapr Docs with examples on how to use the SDK. Actors will be added in a follow up PR. 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

File: daprdocs/content/en/js-sdk-docs/_index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+type: docs
+title: "JavaScript SDK"
+linkTitle: "JavaScript"
+weight: 1000
+description: JavaScript SDK packages for developing Dapr applications
+no_list: true
+---
+The Dapr JS SDK will allow you to interface with the Dapr process that abstracts several commonly used functionalities such as Service-to-Service invocation, State Management, PubSub, and more.
+
```
Comment (XavierGeerinck) [inline daprdocs/content/en/js-sdk-docs/_index.md:10]: I would like to see this a bit more detailed. As to explain what the difference is between a server and client. Maybe we can use the following?

# Dapr JS SDK

* **Client:** Create a Javascript client that can interact with a Dapr Sidecar from your application or other Dapr powered applications.
* **Server:** Create a Server that allows a Dapr Sidecar to interact with your application, forwarding event subscriptions, invokes and more.

Thoughts?

**Comment by XavierGeerinck** [comment]
Other than the 1 comment I added this is looking very nice! Thanks a lot for your amazing contribution, can't wait to merge it 😊

File: daprdocs/content/en/js-sdk-docs/_index.md (hunk 1 above)
Comment (greenie-msft) [reply daprdocs/content/en/js-sdk-docs/_index.md:10]: Good call out. I will add this to the overview page.

**Comment by greenie-msft** [comment]
> Other than the 1 comment I added this is looking very nice! Thanks a lot for your amazing contribution, can't wait to merge it 😊

Please let me know if there is additional feedback. Otherwise it would be good to get this merged into the docs as a first pass! Thank you.

**Review by XavierGeerinck — APPROVED** [review]

**Comment by XavierGeerinck** [comment]
Approved and merged! Thanks a lot!!

**Pull request merged**